experiment/measurer/measure_manager.py

```diff
@@ -755,51 +700,294 @@ def get_pool_args(measurers_cpus, runners_cpus):
 
 
+class LocalMeasureManager(BaseMeasureManager):
+    """Class that holds implementations of core methods for running a measure
+    worker locally."""
+
+    def __init__(self,
+                 experiment: str,
+                 region_coverage=False,
+                 measurers_cpus=None):
+        super().__init__(experiment, region_coverage)
+        self.measurers_cpus = measurers_cpus
+
+    def initialize_queues(
+        self
+    ) -> Tuple[multiprocessing.queues.Queue, multiprocessing.queues.Queue]:
+        return (multiprocessing.Queue(), multiprocessing.Queue())
+
+    def start_workers(self, request_queue: multiprocessing.queues.Queue,
+                      response_queue: multiprocessing.queues.Queue):
+        if not self.measurers_cpus:
+            self.measurers_cpus = multiprocessing.cpu_count()
+            logger.info(
+                'Number of measurer CPUs not passed as argument. using %d',
+                self.measurers_cpus)
+        with multiprocessing.Pool(processes=self.measurers_cpus) as pool:
+            config = {
+                'request_queue': request_queue,
+                'response_queue': response_queue,
+                'region_coverage': self.region_coverage,
+            }
+            local_measure_worker = measure_worker.LocalMeasureWorker(config)
+
+            # Since each worker is going to be in an infinite loop, we dont need
+            # result return. Workers' life scope will end automatically when
+            # there are no more snapshots left to measure.
+            logger.info('Starting measure worker loop for %d workers',
+                        self.measurers_cpus)
+            for _ in range(self.measurers_cpus):
+                _result = pool.apply_async(
+                    local_measure_worker.measure_worker_loop)
+
+    def get_result_from_response_queue(
+            self, response_queue: multiprocessing.queues.Queue):
+        return response_queue.get_nowait()
+
+    def put_task_in_request_queue(
+            self, task: measurer_datatypes.SnapshotMeasureRequest,
+            request_queue: multiprocessing.queues.Queue):
+        request_queue.put_nowait(task)
+
+
+class GoogleCloudMeasureManager(BaseMeasureManager):  # pylint: disable=too-many-instance-attributes
+    """Measurer manager implementation that subscribe and publishes from a
+    Google Cloud Pub/Sub Queue, instead of multiprocessing queue."""
+
+    def __init__(self,
+                 experiment: str,
+                 cloud_project: str,
+                 region_coverage=False,
+                 measurers_cpus=None):
+        super().__init__(experiment, region_coverage)
+        self.project_id = cloud_project
+        self.request_queue_topic_id = f'request-queue-topic-{self.experiment}'
+        self.response_queue_topic_id = f'response-queue-topic-{self.experiment}'
+        self.response_queue_subscription_id = f"""response-queue-subscription-
+            {self.experiment}"""
```

Comment:
nit: How about
```python
self.response_queue_subscription_id = (f'response-queue-subscription-
            {self.experiment}')
```

Reply:
Thanks, I struggled a little bit on how to properly break the string in two lines due to the line-too-long lint error.

I tried the way you suggested and it quite didn't work, but I think I found a solution